### DISCUSSION TITLE
All features merged to main, tag V2.0 created

### DISCUSSION THREAD

**Discussion by CarlosPietrasanta** [CarlosPietrasanta/Tafe230130#10] [Announcements]
Hi team,

Just announcing that Section D of the project has been completed, creating the release branch, merging all the features to this new branch and finally merging all the changes to main.

Also the tag V2.0 has been assigned to this version of the program.

Feel free to continue working on Sections E and F, and please let me know once you completed those steps so we can continue working on Section G.

Thanks,
Carlos.

**Comment by ozdotcom** [comment]
Thanks Carlos,
awesome work.

that last push from me was NOT the updated menu for this repository, I was just having a practice run to my repository in ozdotcom, obviously the global config reset before the push 👎 

Sharryn
